packages/ERTP/package.json

```diff
@@ -53,7 +53,6 @@
   },
   "devDependencies": {
     "@agoric/swingset-vat": "workspace:*",
-    "@agoric/xsnap": "workspace:*",
```

Comment:
interesting. I'm curious what prompted you to notice that this was extraneous

Reply:
The focused install of the planner service tried to build XS and faled